spec/app/ics-020-fungible-token-transfer/README.md

```diff
@@ -620,12 +640,10 @@ function refundTokens(packet: Packet) {
     abortTransactionUnless(false)
   }
 
-  prefix = "{packet.sourcePort}/{packet.sourceChannel}/"
   for token in tokens {
+    onChainDenom = constructOnChainDenom(token.trace, token.denom.base)
     // we are the source if the denomination is not prefixed
-    source = token.trace[0] != prefix
-    onChainDenom = constructOnChainDenom(token.trace, token.denom)
-    if source {
+    if isSource(packet.sourcePort, packet.sourceChannel, token) {
```

Comment:
looks like a logical change. Previously the code did `token.Trace[0] != prefix`, now it does `token.Trace == prefix`

should be previous behaviour

Reply:
think the helper function uses first element of trace so it should be fine?

Reply:
You're right, @colin-axner. The previous logic was correct; I fixed it now.